server/lib/statsd-collector.js

```diff
@@ -65,10 +75,34 @@ function getGenericTags(body) {
     }
   }
 
+  // collect AB choices
+  if (body.ab) {
+    var abTags = [];
+
+    body.ab.forEach(function (item) {
+      if (item.event && item.event === 'choice' && item.experiment && item.choice) {
+        var tagName = 'ab_' + _normalizeTag(item.experiment);
+        var hadChoice = false;
+
+        Object.keys(item.choice).forEach(function (choiceKey) {
+          hadChoice = true;
+          tagName += '_' + _normalizeTag(choiceKey);
```

Comment:
Encoding is hard. I have the hope we can make lookups a little easier. 

From [their docs](http://docs.datadoghq.com/guides/dogstatsd/#datagram-format):

> metric.name should be a String with no colons, bars or @ characters.

Those restrictions are for the name, I imagine the same restrictions, plus a comma, apply to tags, though I'm not 100% sure. Could we use some encoding scheme using other allowed characters like semi-colons, ampersands, or equals?

e.g.,
```
ab_experiment_name;choice_1=choice_1_value&choice_2=choice_2_value;
```

I also remember you pointed me to some docs where I could swear names or tags had a max length of 72 characters, do we need to watch out for that?